weblate/trans/views/edit.py

```diff
@@ -141,7 +141,11 @@ def search(translation, request):
     name = form.get_name() if form_valid else ""
 
     # Grab unit IDs
-    unit_ids = list(allunits.order().values_list("id", flat=True))
+    unit_ids = list(
+        allunits.order_by_request(
+            request.GET.get("sort_by", "").split(",")
```

Comment:
The `request.GET.get("sort_by", "").split(",")` part repeats in every invocation. I'd rather place it into order_by_request and let it accept request as parameter.